lms/djangoapps/instructor/tests/test_ecommerce.py

```diff
@@ -53,6 +66,24 @@ def test_pass_e_commerce_tab_in_instructor_dashboard(self):
         # Coupons should show up for White Label sites with priced honor modes.
         self.assertIn('Coupon Code List', response.content)
 
+    @mock.patch("openedx.core.djangoapps.site_configuration.helpers.get_value", fake_get_value)
```

Comment:
Right, I would add the new `SHOW_ECOMMERCE_REPORTS` setting to [site_configuration_other](https://github.com/edx/edx-platform/blob/d1a3af9460e0381aeb69a42632902370eb7bd505/openedx/core/djangoapps/site_configuration/tests/mixins.py#L36), use `site_configuration_other` for your positive test and the default site for your negative test.